mteb/tasks/Classification/swe/SwedishSentimentClassification.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+N_SAMPLES = 1024
+
+class SwedishSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="SwedishSentimentClassification",
+        description="Dataset of Swedish reviews scarped from various public available websites",
+        reference="https://huggingface.co/datasets/swedish_reviews",
+        dataset={
+            "path": "swedish_reviews",
+            "revision": "105ba6b3cb99b9fd64880215be469d60ebf44a1b",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["validation", "test"],
+        eval_langs=["swe-Latn"],
+        main_score="accuracy",
+        date=("2021-01-01", "2022-01-01"),
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license=None,
```

Comment:
Are we sure it doesn't have a license? If not specified can you please write `"not specified"` ?

Reply:
I couldn't find any. The HF Repo says "Research only" which is not a real license and there was none on github, so I kept it as "Not specified".